webapp/src/DataTransferObject/Shadowing/OrganizationEvent.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace App\DataTransferObject\Shadowing;
+
+class OrganizationEvent implements EventData
+{
+    public function __construct(
+        public readonly string $id,
+        public readonly string $name,
+        public readonly ?string $icpcId,
+        public readonly ?string $formalName,
+        public readonly ?string $country,
+        public readonly ?string $url,
+        public readonly ?string $twitterHashtag,
```

Comment:
If we check this field, shouldn't we also verify the logo?

Reply:
Let's drop it (and URL) since we don't use it